modules/ocf_mirrors/manifests/init.pp

```diff
@@ -45,6 +47,10 @@
       source => 'puppet:///modules/ocf_mirrors/README.html',
       owner  => mirrors,
       group  => mirrors;
+
+    '/usr/local/sbin/record-mirrors-stats':
+      content => template('ocf_mirrors/record-mirrors-stats.py.erb'),
+      mode    => '0644';
```

Comment:
yeah, my mistake 

I modeled this after how it's done in the labstats script, I guess I'll fix this and then fix that as well